vm/registry.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package vm
+
+import "github.com/ava-labs/hypersdk/chain"
+
+type Registry struct {
+	actionRegistry chain.ActionRegistry
+	authRegistry   chain.AuthRegistry
+	outputRegistry chain.OutputRegistry
+}
+
+func NewRegistry(action chain.ActionRegistry, auth chain.AuthRegistry, output chain.OutputRegistry) *Registry {
```

Comment:
Could we define the struct implementing next to the interface in the chain package?

Reply:
done